### ISSUE TITLE
Handle failure of sending an email

### ISSUE THREAD

**Issue by annikakrause** [amosproj/amos2022ss04-digital-identity#101]
## User story

1. As a {HR employee}
2. I want / need {to know if the email to the user of the new created DI has been sent succesfully}
3. So that {can be sure or try e.g. another mailaddress}

## Acceptance criteria

- MailServer: give a useful response after trying to send an email depending on success or failure (not "success" in every case)
- handle response in the backend and refer it to the frontend
- display a message in the frontend depending on the result

## Definition of done (DoD)

- Added only after week 5
- The same for all features
- Here goes the project specific part

## DoD general criteria

- Feature has been fully implemented
- Feature has been merged into the mainline
- All acceptance criteria were met
- Product owner approved features
- All tests are passing
- Developers agreed to release

**Comment by Idontker** [comment]
Done in sprint 5 https://github.com/amosproj/amos2022ss04-digital-identity/commit/8b9cdf974b0592bb76a4d87a3d35fe9834c38272
